README.md

```diff
@@ -27,12 +27,12 @@ Please consult the [LORIS Wiki Setup Guide](https://github.com/aces/Loris/wiki/S
  * LINUX (supported on Ubuntu 14+ and [CentOS 6.5](https://github.com/aces/Loris/blob/master/README.CentOS6.md))
  * Apache2
  * MySQL 5.7 
- * PHP <b>7</b>  
+ * PHP <b>7.2</b>  
```

Comment:
Don't know if this is necessary to change, but the README uses HTML instead of Markdown.

Reply:
That's a good idea. I think out of the scope for this PR though as it's only meant to correct the version information.